### PR TITLE
templater: move empty signature placeholder to user template

### PR DESCRIPTION
**Pull request by yuja**
#2112 

# Checklist

If applicable:
- [x] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [x] I have added tests to cover my changes

### CODE REVIEW

File: cli/src/config/templates.toml [hunk 1]
```diff
@@ -123,13 +123,20 @@ separate(" ",
 
 description_placeholder = '''
   label(if(empty, "empty ") ++ "description placeholder", "(no description set)")'''
+email_placeholder = 'label("email placeholder", "(no email available)")'
```
Comment (avamsi) [inline cli/src/config/templates.toml:126]: Thoughts on calling these "no _ set" to match the description placeholder?

**Review by avamsi — APPROVED** [review]
LGTM (with one optional comment), thanks!

File: cli/src/config/templates.toml [hunk 2]
```diff
@@ -40,7 +40,7 @@ if(root,
           separate(" ",
             format_short_change_id(change_id) ++ if(divergent, "??"),
             if(hidden, "hidden"))),
-        author.username(),
+        if(author.email(), author.username(), email_placeholder),
```
Comment (martinvonz) [inline cli/src/config/templates.toml:43]: Should the two "email" here be "username"? Or what's the thinking behind it otherwise?

Comment (avamsi) [reply cli/src/config/templates.toml:43]: At least I read it as email is the one you actually set and username is just a derivative of that (so I think it makes sense the messaging be around setting email and not username).

Comment (martinvonz) [reply cli/src/config/templates.toml:43]: That makes sense. I just confused "username" with "name"

File: cli/src/config/templates.toml (hunk 1 above)
Comment (yuja) [reply cli/src/config/templates.toml]: Done `s/available/set/`. I think both works, and shorter and consistent output is better.

**Pull request merged**